release-1.8/release_notes_draft.md

```diff
@@ -197,12 +208,19 @@ kind.
 * [alpha] Building on the 1.7 work to allow encryption of resources such as secrets, a mechanism to store resource encryption keys in external Key Management Systems (KMS) was introduced. This complements the original file-based storage and allows integration with multiple KMS. A Google Cloud KMS plugin was added and will be usable once the Google side of the integration is complete.
 
 ### **Cluster Lifecycle**
+
 #### kubeadm
-* [beta] A new `phase` subcommand supports performing only subtasks of the full `kubeadm init` flow. Combined with fine-grained configuration, kubeadm is now more easily consumable by higher-level provisioning tools like kops or GKE. ([#356](https://github.com/kubernetes/features/issues/356), [@luxas](https://github.com/luxas), [@justinsb](https://github.com/justinsb))
 
 * [beta] A new `upgrade` subcommand allows you to automatically upgrade a self-hosted cluster created with kubeadm. ([#296](https://github.com/kubernetes/features/issues/296), [@luxas](https://github.com/luxas))
 
+* [alpha] It is now possible to easily create a self-hosted cluster with `kubeadm init` by enabling the feature (`--feature-gates=SelfHosting=true`) ([#296](https://github.com/kubernetes/features/issues/296), [@luxas](https://github.com/luxas))
```

Comment:
A self-hosted cluster can be easily created with `kubeadm init`. Enable the feature by setting the `SelfHosting` flag to true: `--feature-gates=SelfHosting=true`

Reply:
fixed.